app/utils/error_handling.ts

```diff
@@ -42,7 +43,10 @@ class JavascriptAndNativeErrorHandler {
         }
 
         logWarning('Handling Javascript error', e, isFatal);
-        captureJSException(e, isFatal);
+
+        if (isBetaApp || (!isBetaApp && isFatal)) {
```

Comment:
```suggestion
        if (isBetaApp || isFatal)) {
```

`X || (!X && Y)` is the same as `X || Y`)